README.md

```diff
@@ -1,65 +1,5 @@
 # mipidsi
```

Comment:
IMO the main README should cover both crates. Having an almost empty page when visiting the project on GitHub isn't a great experience for first time users.

The main README should at least contain an overview of what the purpose of the crates is, which is fundamentally the same for both. And also a description of the difference between the two crates to help the user choose the version they want to use. Details that only apply to one of the crates can then be added to the crate specific READMEs.,

Reply:
Thanks for all the suggestions. I've added the top level readme and fixed the licensing and DSI -> DCS issues.

Lmk if there's anything else that still seems off.